### PR TITLE
Map integration

### LINKED ISSUE #27: Map Display
Input: 
 - rings:   [...ringObjects...]
   -   ringObject:   { lat, long, innerDistance, outterDistance, title, color }
 - likes:   [...likeObjects...]
   -   likeObject: { lat, long, color} 
 - displayStats:   { width (in px), height (in px) }

- Expected output: a map object of given size with each `ring` displayed in the given color as well as each `like` dotted throughout. The map should be the dimensions given (don't worry about safeguarding a min or max size, that will be handled on our end)

IF IT SEEMS EASY (otherwise skip):
- A method to add a ring to the map without having to re-render the whole thing 
- A method to delete a ring

### PR DESCRIPTION
**Pull request by GreyMatteOr**
### Description

I integrated Google Maps, swapping out MapBox (let's go over this in detail together). 

You'll notice on the Feed page it creates a ring in the map and centers on it that corresponds to the influence of the post.
I also have the map 'hiding' and 'showing' depending on which page it's on (though it is always there to prevent over-using the API-Key.

### Related Issue
 closes #27 

### How Has This Been Tested?

Clicking around inside the browser

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,19 @@
       @import url('https://fonts.googleapis.com/css2?family=Damion&display=swap');
       @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
     </style>
+    <script>
+      function initMap() {
+        window.earthMap = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 0, lng: 0},
+          zoom: 1
```
Comment (MichaelEWalker87) [inline public/index.html:15]: Ahh I see what you are talking about with the zoom

File: src/Feed/Feed.js [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
-    }
-
-    let { content, createdAt, id, ringMinMax } = data.posts[postIndex]
+    let { content, createdAt, id, latitude, longitude, ringMinMax } = data.posts[postIndex]
     let [min, max] = ringMinMax.slice(1, -1).split(', ').map( char => +char )
 
+    if (needNewRing && earthMapIsLoaded) {
+      if (window.earthRings.length > 0) removeAllRings()
+      let center = {lat: () => latitude, lng: () => longitude}
+      let newRing = addRing( center, min, max )
```
Comment (MichaelEWalker87) [inline src/Feed/Feed.js:26]: Nice!

**Review by MichaelEWalker87 — APPROVED** [review]
Looks good just remove the API Key

File: src/Feed/Feed.js (hunk 2 above)
Comment (GreyMatteOr) [reply src/Feed/Feed.js:26]: Oooh, remind me to take that conditional out. It is redundant any more, leftover from a previous design implementation.

**Pull request merged**